tests/buffered_io.rs

```diff
@@ -0,0 +1,16 @@
+extern crate env_logger;
+extern crate gluon;
+
+use gluon::vm::api::IO;
+use gluon::{new_vm, Compiler};
+
+#[test]
+fn buffered_io() {
+    env_logger::init();
```

Comment:
This will error if there are multiple tests that initialize env_logger so usually I do `let _ = env_logger::try_init()` to ignore the error in case of multiple initializations. It is fine for now though since there is only a single test. And i'd like to move the test to use std.test once that works with `IO` as well so then it becomes a non-issue.